CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ if(CONFIG_HAS_NRFX)
   zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF5340_CPUAPP NRF5340_XXAA_APPLICATION)
   zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF5340_CPUNET NRF5340_XXAA_NETWORK)
   zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF9160        NRF9160_XXAA)
+  zephyr_compile_definitions_ifdef(CONFIG_SOC_COMPATIBLE_NRF52832 NRF52832_XXAA)
```

Comment:
Could you adjust the alignment on the above lines, so that they stey in columns as before ?

Reply:
Done.